client/layout/guided-tours/tours/checklist-about-page-tour.js

```diff
@@ -34,15 +34,15 @@ export const ChecklistAboutPageTour = makeTour(
 			<p>
 				{ translate(
 					'The About Page is often the most visited page on a site. ' +
-						"You might find that it never feels quite done - that's OK. " +
+						'You might find that it never feels quite done - that’s OK. ' +
 						'This is the internet and we can update it as many times as we want. ' +
 						'The key is to just get it started.'
 				) }
 			</p>
 			<p>
 				{ translate(
-					"Let's start by changing the default text with an introduction. " +
-						'Here are some questions to help you out: Who are you and where are you based?' +
+					'Let’s start by changing the default text with an introduction. ' +
```

Comment:
Hi! I've found a possible matching string that has already been translated **1** time:
`translate( 'Let's start by changing the default text with an introduction. Here are some questions to help you out: Who are you and where are you based?Why did you start this site? What can visitors expect to get out of it?' )` *ES Score: 9*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).